server/controller/common/contactus.js

```diff
@@ -0,0 +1,25 @@
+const { postMessage } = require('../../database/queries');
+const { contactusValidation, boomify } = require('../../utilis');
+
+const contactusHandler = async (req, res, next) => {
+  try {
+    const {
+      name, email, message, mobile,
+    } = req.body;
+    await contactusValidation.validateAsync(req.body);
+
+    await postMessage(name, email, message, mobile);
+
+    res.json({
+      status: 200,
+      message: 'message sent succesfully',
```

Comment:
pay attention to typos
```suggestion
      message: 'message sent successfully,
```